src/extension.ts

```diff
@@ -63,3 +80,77 @@ const getComponentName = (): Thenable<string | undefined> =>
         : "Please enter a valid component name";
     },
   });
+
+/**
+ * Check availability of Angular CLI on the command line
+ * @param directory Directory which will use the Angular CLI
+ */
+const isAngularCliAvailable = (directory: string): boolean => {
+  try {
+    execSync("ng --version", { cwd: directory });
+    return true;
+  } catch {
```

Comment:
(Optional) Maybe logging the reason is still a good idea

Reply:
👍